scripts/docs-collator/utils/rendering.py

```diff
@@ -119,13 +122,13 @@ def replace_relative_links_with_github_links(repo, content, relative_path=None):
         updated_link = link
 
         # remove leading './' or '/'
-        if link.startswith('./'):
-            updated_link = updated_link.replace('./', '', 1)
-        elif link.startswith('/'):
+        if link.startswith('/'):
             updated_link = updated_link.replace('/', '', 1)
-
-        if relative_path:
-            updated_link = f"{relative_path}/{updated_link}"
+        else:
+            if link.startswith('./'):
+                updated_link = updated_link.replace('./', '', 1)
+            if relative_path:
+                updated_link = f"{relative_path}/{updated_link}"
```

Comment:
If you are using python 3.9+, this simplifies to:
```suggestion
        updated_link = updated_link.removeprefix("./")
        updated_link = updated_link.removeprefix("/")
        if relative_path:
            updated_link = f"{relative_path}/{updated_link}"
```

Reply:
`relative_path` is the path from the root of the repo to the "current" directory. So if reading `/module/a/README.md`, `relative path would be `module/a`. Now say the README has a link `/module/other/examples/complete`. What would you want `updated_link` to be?

Reply:
I think I just misunderstood the intent of the path concatenation. I'll just resolve this